addons/snailmail/static/src/js/thread_widget.js

```diff
@@ -33,10 +33,8 @@ ThreadWidget.include({
     /**
      * @override
      */
-    render: function (thread, options) {
```

Comment:
Why changing the arguments?

Reply:
Unused arguments. The linter can't know that this method is inherited from other. But is not a problem because `arguments` array is filled anyway.
This happens in OCA, i dont know if i should apply the same rules here.

Reply:
No, please limit here the changes to do to the minimum required for fixing the problem. If this is only a linter improvement, starting with, Odoo has other linter (or no linter at all), and anyways, this kind of improvements is not done on stable branches.

Reply:
Changes done